backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -1053,8 +1053,12 @@ class FileInfoAbstractSchema(marshmallow.Schema):
         example=1024,
         allow_none=True,
     )
-    pdf_available = marshmallow.fields.Bool(
-        description="Is pdf version of file available ?",
+    has_pdf_preview = marshmallow.fields.Bool(
+        description="true if a pdf preview is available or false",
```

Comment:
if you put JPEG, put PDF. Or put both in lowercase

Reply:
done in lowercase.